Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -150,7 +150,10 @@ class ActivityStreamPanel: UICollectionViewController, HomePanel {
     func applyTheme() {
         collectionView?.backgroundColor = UIColor.theme.browser.background
         topSiteCell.collectionView.reloadData()
-        reloadAll()
+        if let collectionView = self.collectionView {
+            collectionView.reloadItems(at: collectionView.indexPathsForVisibleItems)
+            collectionView.reloadData()
```

Comment:
TIL: collectionView.reloadData() == collectionView.reloadItems(at: collectionView.indexPathsForVisibleItems)
They are equivalent according to
https://developer.apple.com/documentation/uikit/uicollectionview/1618078-reloaddata

`reloadItems` is for updating part of the visible screen